[2025-01-04T08:09:48.943+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: insert_data.md_currency_d scheduled__2025-01-03T00:00:00+00:00 [queued]>
[2025-01-04T08:09:48.960+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: insert_data.md_currency_d scheduled__2025-01-03T00:00:00+00:00 [queued]>
[2025-01-04T08:09:48.961+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2025-01-04T08:09:49.444+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): md_currency_d> on 2025-01-03 00:00:00+00:00
[2025-01-04T08:09:49.462+0000] {standard_task_runner.py:60} INFO - Started process 97 to run task
[2025-01-04T08:09:49.467+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'insert_data', 'md_currency_d', 'scheduled__2025-01-03T00:00:00+00:00', '--job-id', '308', '--raw', '--subdir', 'DAGS_FOLDER/insert_data.py', '--cfg-path', '/tmp/tmp0d7sphm9']
[2025-01-04T08:09:49.472+0000] {standard_task_runner.py:88} INFO - Job 308: Subtask md_currency_d
[2025-01-04T08:09:49.558+0000] {task_command.py:423} INFO - Running <TaskInstance: insert_data.md_currency_d scheduled__2025-01-03T00:00:00+00:00 [running]> on host 403e7aadd7ee
[2025-01-04T08:09:49.689+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='agushchin' AIRFLOW_CTX_DAG_ID='insert_data' AIRFLOW_CTX_TASK_ID='md_currency_d' AIRFLOW_CTX_EXECUTION_DATE='2025-01-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-03T00:00:00+00:00'
[2025-01-04T08:09:49.729+0000] {base.py:83} INFO - Using connection ID 'postgres-db' for task execution.
[2025-01-04T08:09:49.814+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-01-04T08:09:49.831+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=insert_data, task_id=md_currency_d, execution_date=20250103T000000, start_date=20250104T080948, end_date=20250104T080949
[2025-01-04T08:09:49.883+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-01-04T08:09:49.929+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
